modules/gdscript/editor/script_templates/CharacterBody3D/basic_movement.gd

```diff
@@ -5,29 +5,29 @@ extends _BASE_
 
 const SPEED = 5.0
 const JUMP_VELOCITY = 4.5
+const FRICTION = 10.0
 
-# Get the gravity from the project settings to be synced with RigidBody nodes.
+# Get the gravity from the project settings, to allow syncing with RigidBody3D nodes.
```

Comment:
I don't think this wording is an improvement. It makes it sound like syncing is an extra thing the user has to do and this code allows it. However, this code is what handles the syncing.

Reply:
Yeah, I'm still not happy with that either. How about this:

> Get the global gravity from the project settings. Rigid bodies also use this value.

Reply:
I think that's better than before. How about this?

> Get the global gravity from the project settings. RigidBody3D nodes also use this value.

Reply:
Works for me.